src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -944,7 +965,9 @@ export default {
 			// Query to update basket state
 			this.updateBasketState();
 			// TEMPORARY: Obstruct ability to click the "Checkout" button on the loan card to prevent redirect
-			this.$refs.loandisplayref.loadingLoans = true;
+			if (this.$refs.loandisplayref) {
+				this.$refs.loandisplayref.loadingLoans = true;
```

Comment:
Oh man! accessing refs that don't exist cause "phantom" errors that don't bubble up. Guarding these before use allowed me to convert the loan displays to v-if's instead of v-show's.